docs/src/main/sphinx/develop/delete-and-update.rst

```diff
@@ -24,11 +24,11 @@ methods to get a "rowId" column handle; to start the operation; and to finish th
   ``UpdatablePageSource`` ``deleteRows()`` or ``updateRows()`` method.  Those methods persist
   the deletions or updates in the underlying data store.
 * When all the pages for a specific split have been processed, the Trino engine calls
-  ``UpdatablePageSource.finish()``, which returns a ``Collection<Slice>`` of "fragments"
+  ``UpdatablePageSource.finish()``, which returns a ``Collection<Slice>`` of fragments
   representing connector-specific information about the rows processed by the calls to
   ``deleteRows`` or ``updateRows``.
 * When all pages for all splits have been processed, the Trino engine calls ``ConnectorMetadata.finishDelete()`` or
-  ``finishUpdate``, passing a collection containing all the "fragments" from all the splits.  The connector
+  ``finishUpdate``, passing a collection containing all the fragments from all the splits.  The connector
```

Comment:
can we remove the double space and wrap this?

Reply:
That opened a small can of worms. I ran the Vale linter on it and found some other minor typos.